202009.0/c0b736b5-67db-490e-b93b-3962f3950c9b.md

```diff
@@ -217,6 +217,10 @@ Provides information and functionality necessary for sorting the results.
 Adds a spelling correction suggestion to search results.
 Use `\Spryker\Client\SearchElasticsearch\Plugin\QueryExpander\SpellingSuggestionQueryExpanderPlugin` to let Elasticsearch provide the “did you mean” suggestions for full-text search typos. The suggestions are collected from the `suggestion_terms` field of `page` index map, so you need to make sure to store only those information inside this field that you’d like to use for this purpose. The necessary result formatter for this plugin is `\Spryker\Client\SearchElasticsearch\Plugin\ResultFormatter\SpellingSuggestionResultFormatterPlugin`
 
+#### Fuzzy search (query)
+Lookups for products even if a customer makes typos and spelling mistakes in a search query.
+Use `\Spryker\Client\SearchElasticsearch\Plugin\QueryExpander\FuzzyQueryExpanderPlugin` to let Elasticsearch add the `"fuzziness": "AUTO"` parameter to any match query that is created as the suggested search.
```

Comment:
@lenadoc what if it's not integrated into next release of B2C? can we mention it here?

Reply:
@matweew Is it integrated into B2B? If so, you can add a note that this is valid for the B2B Demo Shop only.

Reply:
@olehnovatskyi-spryker Please add this note in the appropriate place: "This is valid for Master Suite only and has not been integrated into B2B/B2C Suites yet."